src/knotificationmanager.cpp

```diff
@@ -291,6 +291,12 @@ void KNotificationManager::close(int id, bool force)
 
 int KNotificationManager::notify(KNotification *n)
```

Comment:
I have rewritten the implementation of Notify to fully support the kdeconnect's notification calls.